cellxgene_resources/curation_qa.ipynb

```diff
@@ -455,19 +425,24 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "prop = 'sample_id'\n",
+    "prop = 'donor_id'\n",
     "\n",
     "results = []\n",
+    "csv = '../ref_files/10X_barcode_table.csv'\n",
```

Comment:
This is run from cellxgene_resources/, which is where ref_files/ exists.
The `..` should be removed - it fails to find the file other wise (`..` tells it to look in one parent directory up)